addons/web/static/src/js/fields/field_utils.js

```diff
@@ -200,10 +200,16 @@ function formatFloatFactor(value, field, options) {
  * instead of 0.25.
  *
  * @param {float} value
+ * @param {Object} [field]
+ *        a description of the field (note: this parameter is ignored)
+ * @param {object} [options]
+ * @param {boolean} [options.noLeadingZeroHour] if true, format as %1d:%02d
+ *        otherwise, format as %02d:%02d
```

Comment:
give an example instead of the format pattern? It would be straightforward to understand then...

Reply:
also, this should be tested :D